scripts/manager_time.lua

```diff
@@ -18,23 +18,13 @@ function onInit()
 end
 
 --- Timer Functions
-function setStartTime(rActor, sFirst)
-	-- Debug.console("setStartTime called; " .. sFirst .."");
-	local nodeActor = rActor;
-	nStartTime = getCurrentDateinMinutes(rActor);
-	-- Debug.console("setStartTime; nStartTime =", nStartTime);
+function setStartTime(nodeActor)
+	local nStartTime = getCurrentDateinMinutes();
 	DB.setValue(nodeActor, "starttime", "number", nStartTime);
-	Debug.console("setStartTime", rActor, sFirst, nStartTime, DB.getValue(nodeActor, "starttime"));
-	-- Debug.console("setStartTime; DB.setValue(nodeActor, " .. sFirst .. ".starttime, number, " .. nStartTime .. ") = ", DB.setValue(nodeActor, "" .. sFirst .. ".starttime", "number", nStartTime));
 end
```

Comment:
Cleaned up this function.  The arg coming in is a node, so I eliminated the redundant assignment and extraneous arg to the current date call.  Got rid of the global use too.

Reply:
Yeah! I've noticed there are a lot of globals in ClockAdjuster. They don't seem to be fouling anything up yet, but not ideal!